lstchain/io/tests/test_dataselection.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+from lstchain.io import DataSelection, DataBinning
```

Comment:
I don't understand what you are testing here / what `DataSelection` is supposed to do.

Reply:
I was not sure whether I can test the Components `DataSelection` and `DataBinning` and this seems pointless to me, but I thought, every `Component` should have a test, so I just test whether the functions work, with the given value_traitlets.

Reply:
You didn't implemnt any actual logic into these components.  What are they supposed to do?

I would assume an event selector get's a table of events and returns either a mask or a new table with only the selected events.

I don't see any of that here.

Reply:
Ah yes! Sorry for that! I naïvely just kept them to intake selection and binning parameters, and almost doing nothing. I will implement a proper event selector, so that both the Tools can use it.